CHANGELOG.md

```diff
@@ -6,110 +6,134 @@ Contributors to this file, please follow the guidelines on [keepachangelog.com](
 
 For reference, the possible headings are:
 
-* **New Feature** for new features.
-* **Improvement** for changes in existing functionality.
-* **Bug** for any bug fixes.
+* **New Features* for new features.
+* **Improvements** for changes in existing functionality.
+* **Bugs** for any bug fixes.
 * **External Contributors** to list contributors outside of SonarSource SA.
 * **Notes**
 
 
 ## [Unreleased]
 
+## [3.8]
+
+### New Features
+* [#151](https://github.com/SonarSource/sonarlint-visualstudio/issues/151) - Enable connected mode for .Net Core projects
+
+### Improvements
+* [#435](https://github.com/SonarSource/sonarlint-visualstudio/issues/435) - Upgrade additional analyzers with the latest versions
+* [#451](https://github.com/SonarSource/sonarlint-visualstudio/issues/451) - Update SonarLint description for the marketplace
+
+### Bugs
+* [#440](https://github.com/SonarSource/sonarlint-visualstudio/issues/440) - OptOut button does not work
+
+
+## [3.7]
+
+### Bugs
+* [#395](https://github.com/SonarSource/sonarlint-visualstudio/issues/395) - Binding the solution to SonarQube before all projects have finished to load result in weird error
+* [#413](https://github.com/SonarSource/sonarlint-visualstudio/issues/413) - Opening a solution with duplicate solution folder names crashes VS
+
+### Improvements
+* [#347](https://github.com/SonarSource/sonarlint-visualstudio/issues/347) - "Connect" option is not available unless a solution is open
```

Comment:
Let's rename this ticket to actually state the solution: "Connect option should be available only after a solution is loaded".

Reply:
I don't think we should put the solution in ticket names. That would mean a lot of renaming after tickets are addressed.

But agree for renaming in this instance. The name is a bit confusing, your suggestion is much better.